### PR TITLE
added `Wikipedia` retriever

### PR DESCRIPTION
**Pull request by leo-gan**
- added `Wikipedia` retriever. It is effectively a wrapper for `WikipediaAPIWrapper`. It wrapps load() into get_relevant_documents()
- sorted `__all__` in the `retrievers/__init__`
- added integration tests for the WikipediaRetriever
- added an example (as Jupyter notebook) for the WikipediaRetriever

### CODE REVIEW

**Review by hwchase17 — COMMENTED** [review]
will this work out of the box in a question answer chain? can you show an example of what that would look like?

**Comment by leo-gan** [comment]
> 

@hwchase17 I've added an example with ConversationalRetrievalChain to the notebook. I also added a default limit size to the Document content. 
Note: I've commented out a line in the example
`# "How big is Wikipédia en français?",` because if fails. I'm not sure it fails in my code. Could you, please, check it?

File: langchain/utilities/wikipedia.py [hunk 1]
```diff
@@ -104,12 +106,12 @@ def load(self, query: str) -> List[Document]:
...
         See
 
-        Returns: a list of documents with the document.page_content in PDF format
+        Returns: a list of documents.
 
         """
         page_titles = self.wiki_client.search(query[:WIKIPEDIA_MAX_QUERY_LENGTH])
         docs = []
-        for page_title in page_titles[: self.top_k_results]:
+        for page_title in page_titles[: self.doc_content_chars_max]:
```
Comment (dev2049) [inline langchain/utilities/wikipedia.py:114]: is page_titles a string or a list of things? was this line supposed to be changed?

Comment (leo-gan) [reply langchain/utilities/wikipedia.py:114]: A good catch! Thanks! Yes, it should be `top_k_results`.

**Comment by dev2049** [comment]
this is awesome, thanks @leo-gan!

**Pull request merged**